rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -65,17 +66,21 @@
  * @since 22.05.2014
  */
 public abstract class SystemProperties {
-    public static final String DEFAULT_BIND_IP = "::";
+    private static final String DEFAULT_BIND_ADDRESS = "::";
+    private static final int DEFAULT_RPC_PORT = 4444;
     private static Logger logger = LoggerFactory.getLogger("general");
 
     public static final String PROPERTY_DB_DIR = "database.dir";
-    public static final String PROPERTY_LISTEN_PORT = "peer.listen.port";
+    public static final String PROPERTY_PEER_PORT = "peer.port";
     public static final String PROPERTY_PEER_ACTIVE = "peer.active";
     public static final String PROPERTY_DB_RESET = "database.reset";
     // TODO review rpc properties
     public static final String PROPERTY_RPC_ENABLED = "rpc.enabled";
     public static final String PROPERTY_RPC_PORT = "rpc.port";
     public static final String PROPERTY_RPC_CORS = "rpc.cors";
+    public static final String PROPERTY_RPC_ADDRESS = "rpc.address";
+    public static final String PROPERTY_PUBLIC_IP = "public.ip";
+    public static final String PROPERTY_BIND_ADDRESS = "bind.address";
```

Comment:
if `peer` is already used as a prefix then I expect `bind.address` to have that prefix if `peer.port` has the prefix.

Reply:
The same as public ip, will remain outside peer configuration